src/overlays/actors/ovl_En_Kusa2/z_en_kusa2.h

```diff
@@ -4,14 +4,81 @@
 #include "global.h"
 
 struct EnKusa2;
+struct EnKusa2UnkBssSubStruct;
 
 typedef void (*EnKusa2ActionFunc)(struct EnKusa2*, GlobalContext*);
+typedef void (*EnKusa2BssFunc)(struct EnKusa2UnkBssSubStruct*);
+
+#define ENKUSA2_GET_1(thisx) (((thisx)->params) & 1)
+#define ENKUSA2_GET_7F00(thisx) (((((thisx)->params) >> 8) & 0x7F) & 0xFF)
```

Comment:
Does it still match with a cast to `u8` instead of the explicit `& 0xFF` at the end? If it does that would be my preference, but up to you either way.